src/v3/src/transformer/phone/__snapshots__/phoneChallengeTransformer.test.ts.snap

```diff
@@ -71,7 +71,7 @@ Object {
       Object {
         "contentType": "subtitle",
         "options": Object {
-          "content": "oie.phone.verify.sms.codeSentText <span class=\\"strong no-translate\\">+1 XXX-XXX-4601</span>. oie.phone.verify.enterCodeText",
+          "content": "oie.phone.verify.sms.codeSentText <span class=\\"strong no-translate\\">+1 XXX-XXX-4601.</span> oie.phone.verify.enterCodeText",
```

Comment:
that's odd, is there any way to change that behavior?

Reply:
The assert English leaks function is what makes that determination, any character (that is non empty) found that doesn't have the ok-pl tokens, it fails. 

The only real way to fix it is if we included the period inside of the translation key's value (this way OK-pl will convert it properly).

Reply:
Yea, however, we should probably include the period inside of the translation text, tbh this entire should should be revisited because they are concatenating different keys together which is an anti-pattern w.r.t. translation. I can file a bug for this. That would ultimately resolve all of this 🙏

Reply:
Agreed re: don't concatenate keys. We can improve this in the JIRA you filed, thanks!

Reply:
🙏 https://oktainc.atlassian.net/browse/OKTA-595114